InfiniteRecharge/src/main/java/frc/robot/Constants.java

```diff
@@ -11,9 +11,14 @@
  * Add your docs here.
  */
 public class Constants {
+    
 
 	public static final int kRobotWranglerSparkCanId = 148;
 	public static final boolean kUseRobotWranglerNeoEncoder = true;
 	public static final int kRobotWranglerReverseLimitDio = 1114;
 	public static final int kRobotWranglerForwardLimitDio = 2056;
+	public static final int kLeftDriveACanID = 1;
+	public static final int kLeftDriveBCanID = 1;
+	public static final int kRightDriveACanID = 1;
+	public static final int kRightDriveBCanID = 1;
```

Comment:
lolwut

Reply:
That is for the AutoCan2000! I have fixed it now.